dftimewolf/lib/processors/stackdriver_timesketch.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+"""Processes Stackdriver logs for loading into Timesketch."""
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import tempfile
+import json
+
+from dftimewolf.lib.module import BaseModule
+from dftimewolf.lib.containers import StackdriverLogs
+
+
+class StackdriverTimesketch(BaseModule):
+  """Transforms Stackdriver logs for Timesketch."""
+
+  def __init__(self, state):
+    super(StackdriverTimesketch, self).__init__(state)
+
+  def setup(self, *args, **kwargs):
+    """Sets up necessary module configuration options."""
+
+  def cleanup(self):
+    """Cleans up module output to prepare it for the next module."""
+
+  def _process_log_line(self, log_line, query, project_name):
```

Comment:
I wonder if the log parsing logic would not be better implemented in an external module (like `utils` or something similar), so that they can eventually be reused (maybe?) and don't pollute the dftimewolf module class.

Reply:
This is in a separate processor module, so we can extract it somewhere if we need to in the future, but it's pretty specific to Stackdriver->Timesketch mapping.